ui/app/styles/components/header.scss

```
@@ -0,0 +1,13 @@
+/**
+ * Copyright (c) OpenBao
```

```suggestion
 * Copyright (c) 2024 OpenBao a Series of LF Projects, LLC
```

This was our preferred Copyright header for new files.